src/integration-test/java/uk/gov/hmcts/juror/api/moj/controller/CompleteServiceControllerITest.java

```diff
@@ -73,12 +73,37 @@ public void setUp() throws Exception {
         httpHeaders.setAccept(Collections.singletonList(MediaType.APPLICATION_JSON));
     }
 
+
+    private void validateJurorWasCompleted(LocalDate completionTime, String jurorNumber, String poolNumber,
+                                           boolean isDismissal) {
+        JurorPool jurorPool = jurorPoolRepository.findByJurorJurorNumberAndPoolPoolNumber(jurorNumber, poolNumber);
+        assertEquals(true, jurorPool.getIsActive(),
+            "Juror pool should be inactive");
```

Comment:
Message states pool should be inactive but assertion is to see if the juror pool is active. Either the message needs updating or the true value should be flipped to false.

Reply:
Updated